exo-run/features/steps/steps-given.ls

```diff
@@ -18,4 +19,4 @@ module.exports = ->
                                        cwd: @app-dir,
                                        stdout: dim-console.process.stdout
                                        stderr: dim-console.process.stderr)
-      done!
+      wait 10_000, done
```

Comment:
The first alternative that comes to mind, though I'm not sure its a great option by any means, is that we have a docker image built `FROM` something tiny like the `alpine` image (5MB). that simply prints a message such as `all services online`, and `depends_on` all other services of the application so that it would only be run after all other containers are ready.

I'll keep looking into better options.